src/gui/views/list.rs

```diff
@@ -428,17 +428,17 @@ impl List {
                 {
                     column![
                         control_panel,
-                        packages_scrollable,
-                        description_panel,
-                        action_row,
+                        container(unavailable).height(Length::Fill).center_y(),
                     ]
                     .width(Length::Fill)
                     .spacing(10)
                     .align_items(Alignment::Center)
                 } else {
                     column![
                         control_panel,
-                        container(unavailable).height(Length::Fill).center_y(),
+                        packages_scrollable,
+                        description_panel,
+                        action_row,
                     ]
```

Comment:
Conflict resolution issue. I'm pushing a fix.

Reply:
Fixed in [9105783](https://github.com/0x192/universal-android-debloater/pull/673/commits/9105783fb547969e9821cd3c88d621128bc656c7)